modules/upi-provisioning-new-hosts-in-a-upi-cluster.adoc

```diff
@@ -20,45 +20,156 @@ To provision bare-metal hosts to the cluster by using the BMO, you must set the
 
 .Procedure
 
-. Create the `Secret` CR and the `BareMetalHost` CR.
+. Create a configuration file for the bare-metal node. Depending if you use either a static configuration or a DHCP server, choose one of the following example `bmh.yaml` files and configure it to your needs by replacing values in the YAML to match your environment:
 
-.. Save the following YAML in the `bmh.yaml` file:
+.Static configuration `bmh.yaml`
```

Comment:
The code example is not rendering correctly in the preview:

![image](https://github.com/user-attachments/assets/4c80fa4f-9e3d-4b38-bfed-39627c78be56)

And it ripples all the way down to the _Create the bare-metal node by running the following command:_ step.